packages/office-ui-fabric-react/src/components/DocumentCard/DocumentCardActivity.base.tsx

```diff
@@ -1,37 +1,47 @@
 import * as React from 'react';
-import { BaseComponent, css } from '../../Utilities';
-import { IDocumentCardActivityProps, IDocumentCardActivityPerson } from './DocumentCard.types';
```

Comment:
See in here the renaming preserved the history and you get a nice diff. I wonder why it didn't work on `DocumentCardActions` 🤔